performance/load_test_performance_analysis.py

```diff
@@ -0,0 +1,323 @@
+import os
+import sys
+import json
+import pandas as pd
+from datetime import datetime
+import time
+import random
+import logging
+from concurrent.futures import ThreadPoolExecutor
+import xml.dom.minidom
+import xml.etree.ElementTree as ET
+import argparse
+import typing as t_
+
+from smsdk.client import Client
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+# Function to read configuration from config.json
+def read_config() -> t_.Dict[str, t_.Any]:
+    config = {}
+    try:
+        with open("config.json", "r") as f:
```

Comment:
It would be nice to have an option to specify a different config file.  This fine for default behaviour

Reply:
Done.